internal/crawler/crawler.go

```diff
@@ -106,10 +127,49 @@ func htmlHandler(e *colly.HTMLElement) {
 	}
 }
 
-func xmlHandler(e *colly.XMLElement) {
-	err := e.Request.Visit(e.Text)
-	if err != nil && !isForbiddenURLError(err) {
-		log.Error().Err(err).Str("link", e.Text).Msg("Error attempting to visit link")
+func sitemapXmlHandler(e *colly.XMLElement) {
+	nodes, _ := xmlquery.QueryAll(e.DOM.(*xmlquery.Node), "//sitemap")
+	num_sitemaps = len(nodes)
+
+	xmlquery.FindEach(e.DOM.(*xmlquery.Node), "//sitemap", func(i int, child *xmlquery.Node) {
+		err := e.Request.Visit(child.SelectElement("loc").InnerText())
+		if err != nil && !isForbiddenURLError(err) {
+			log.Error().Err(err).Str("link", e.Text).Msg("Error attempting to visit link")
+		}
+
+		es = append(es, entry{
+			val: child.SelectElement("lastmod").InnerText(), key: child.SelectElement("loc").InnerText()})
```

Comment:
This would be easier to read if you broke the different attributes into separate lines (in the same way you have here: https://github.com/alphagov/govuk-mirror/pull/67/files#diff-4b16ae3250117a18bb13dadcb64ff113bb330dc0f124f81f9a5fb2d6b16c638bR154-R156